pkg/lib/packagekit.js

```diff
@@ -250,7 +250,7 @@ export function cancellableTransaction(method, arglist, progress_cb, signalHandl
                 progress_data.waiting = allow_wait_status && (status === Enum.STATUS_WAIT || status === Enum.STATUS_WAITING_FOR_LOCK);
                 if ("AllowCancel" in props)
                     progress_data.cancel = props.AllowCancel ? cancel : null;
-                if ("Percentage" in props && props.Percentage <= 100)
+                if ("Percentage" in props?.Percentage <= 100)
```

Comment:
This is wrong. I wouldn't rely on the unintuitive semantics of JS to compare `undefined` against a number, so let's keep the old code.

Reply:
Done